Kinvey/KinveyTests/MultiInsertSpec.swift

```diff
@@ -0,0 +1,1492 @@
+                        expect(entity?.entityId).toNot(beNil())
+                    }
+                    it("should send PUT with a sngle item with _id") {
+                        var entity = kinveySave(dataStore: networkDataStore, entity: Person { $0.name = "Victor" }).entity
+                        expect(entity).toNot(beNil())
+                        expect(entity?.entityId).toNot(beNil())
+                        expect(entity?.name).to(equal("Victor"))
+                        
+                        guard let entityId = entity?.entityId else {
+                            return
+                        }
+                        guard let entityUnwrapped = entity else {
+                            return
+                        }
+                        
+                        entityUnwrapped.name = "Hugo"
+                        entity = kinveySave(dataStore: networkDataStore, entity: entityUnwrapped).entity
+                        expect(entity).toNot(beNil())
+                        expect(entity?.entityId).toNot(beNil())
+                        expect(entity?.name).to(equal("Hugo"))
+                        
+                        entity = kinveyFind(dataStore: networkDataStore, id: entityId).result
+                        expect(entity).toNot(beNil())
+                        expect(entity?.entityId).toNot(beNil())
+                        expect(entity?.name).to(equal("Hugo"))
+                    }
+                    it("should accept an array of items") {
+                        let entity = kinveySave(dataStore: networkDataStore, entity: Person { $0.name = "Victor" }).entity
+                        expect(entity).toNot(beNil())
+                        expect(entity?.entityId).toNot(beNil())
+                        expect(entity?.name).to(equal("Victor"))
+                        
+                        guard let entityUnwrapped = entity else {
+                            return
+                        }
+                        
+                        entityUnwrapped.name = "Victor 2"
+                        let error = kinveySaveMulti(dataStore: networkDataStore, entities: entityUnwrapped, Person { $0.name = "Hugo" }).error
+                        expect(error).toNot(beNil())
+                        expect(error as? Kinvey.Error).toNot(beNil())
+                        guard let kinveyError = error as? Kinvey.Error else {
+                            return
+                        }
+                        switch kinveyError {
+                        case .featureUnavailable(let debug, let description):
+                            expect(debug).to(equal("Inserting multiple entities is not available in this Kinvey API version"))
+                            expect(description).to(equal("Requested functionality is unavailable in this API version."))
+                        default:
+                            fail(kinveyError.localizedDescription)
```

Comment:
but the `restApiVersion` needs to be set manually to `5` otherwise you will receive this message, so this test is indeed necessary

Reply:
I meant that the test is called 'should accept an array of items', however, an error is returned for multi-insert. I though you had done the test to comply with my list, even though it is not working the same way. Also, isn't there basically the same test a bit above, that confirms with API version other than 5, an error is returned?

Reply:
yes, that's my current understanding of the feature. If the user is trying to call `save()` passing as a parameter an array of items AND the `REST API version` is not `5 or above`, the user will receive an error, since the backend will return an error saying the feature is not available in the API version used. Did I miss something? Sorry, but I'm not sure if I understood correctly what you are saying.

Reply:
ohhh, I think I got you are saying... you are suggesting that the SDK should perform N (number of items in the array) single requests if the REST API version is not 5 and above? Is that what you are saying?

Reply:
@heyzooi - Just had a conversation offline with @Martin-Hristov and the issue with this test is that it is redundant, and therefor unnecessary. Since you are performing the same API version check and returning the same error in this case as in the test above, we do not need this test.

Reply:
ignoring this test for now adding and `x` to the test, like `xit()` instead of `it()`